features/search_functionality.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Search Functionality
+  As a user,
+  In order to filter the images
+  I should be able to search
+
+Background:
+  Given that i am on the landing page
+  And there is a field "search_field"
+  Given The following photos exist
+    | title   | price   | image_path                              |
+    | Sporty1 | 30      | http://lorempixel.com/400/200/sports/1/ |
+    | Sporty2 | 40      | http://lorempixel.com/400/200/sports/2/ |
+    | Sporty3 | 50      | http://lorempixel.com/400/200/sports/3/ |
+    | Sporty4 | 60      | http://lorempixel.com/400/200/sports/4/ |
+
+Scenario:
```

Comment:
Please consider adding a description after the scenario. This will make it easier to identify what the scenario is testing are doing when running the test.  
In this case maybe something like this:
Scenario: Search from the landing page [Happy path]